src/CoreGraphics/CGContext.cs

```diff
@@ -579,6 +579,15 @@ public void EOClip ()
 		}
 
 		[DllImport (Constants.CoreGraphicsLibrary)]
+		extern static void CGContextResetClip (/* CGContextRef */ IntPtr c);
```

Comment:
Is this avail on macOS, tvOS and watchOS?
Needs avail attribute for intro tests